### ISSUE TITLE
Port getfileinfo(full) from jscript

### ISSUE THREAD

**Issue by regorxxx** [TheQwertiest/foo_spider_monkey_panel#180]
Would it be possible to implement this?
https://marc2k3.github.io/jscript-panel/docs/interfaces/IMetadbHandle/#getfileinfofull_info

Have found chromaprint fingerprint matching becomes impossible for big libaries.

The full raw fingerprints must be cached by foobar at init (otherwise we get the '.' value), but then it easily takes 2GB of ram for 70 K tracks XD. I tried to alleviate the problem creating a reverse index dictionary (cached in a file), which takes 1GB... but I have to load the raw fingerprints to create it first, thus reaching +3GB limit. 

I had to totally disable the chromaprint logic within my scripts due to memory allocation crashes. 

The ram limitation would be easily solved by reading directly from files.  It would also be obviously useful for other use cases, like lyrics, etc.

**Comment by marc2k3** [comment]
Won't be necessary with fb2k v2. Using fb.TitleFormat or the existing GetFileInfo will fetch oversized tags in full.

**Comment by regorxxx** [comment]
Yep, just thought it when reading the thread at HA. Peter wrote it a day later hahahaha

**Comment by regorxxx** [comment]
Well after all this time, it will be fine on v2 it seems since tag caching will use the entire tag.

But v1 would still appreciate the method.
